src/wallet/mod.rs

```diff
@@ -5414,4 +5417,40 @@ pub(crate) mod test {
         // ...and checking that everything is fine
         assert_fee_rate!(psbt, details.fee.unwrap_or(0), fee_rate);
     }
+
+    #[cfg(feature = "test-hardware-signer")]
+    #[test]
+    fn test_create_signer() {
+        use hwi::{types, HWIClient};
+
+        use crate::wallet::hardwaresigner::signer::HWISigner;
+
+        let devices = HWIClient::enumerate().unwrap();
+        let client = HWIClient::get_client(
+            devices.first().expect(
+                "No devices found. Either plug in a hardware wallet, or start a simulator.",
+            ),
+            true,
+            types::HWIChain::Regtest,
+        )
+        .unwrap();
```

Comment:
When running the test I am getting an error at this line 
```
thread 'wallet::test::test_create_signer' panicked at 'called `Result::unwrap()` on an `Err` value: PyErr("TypeError: get_client() takes from 2 to 4 positional arguments but 5 were given")', src/wallet/mod.rs:5436:10
```

Am I doing something wrong??

Reply:
Make sure you have hwi library >=2.1.1. The latest version of hwi through pip should work fine.

Reply:
Yup.. That was problem.. Working now..